app/assets/stylesheets/show.scss

```diff
@@ -0,0 +1,76 @@
+@import "foundation_and_overrides";
+@import "index";
+
+.art-details {
+  margin: auto;
+  margin-top: 20px;
+  margin-bottom: 35px;
+  display: block;
+  width: 50%;
+  background-color: #088da5;
```

Comment:
Color literals like `#088da5` should only be used in variable declarations; they should be referred to via variable everywhere else.